scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -283,8 +283,20 @@ class ScanCode(name: String, config: ScannerConfiguration) : LocalScanner(name,
         }
     }
 
+    private fun getFileCount(result: JsonNode): Int {
+        // Handling for ScanCode 2.9.8+
+        result.get("headers")?.forEach { header ->
+            header.get("extra_data")?.get("files_count")?.let {
+                return it.intValue()
+            }
+        }
+
+        // Backwards compatibility for ScanCode pre 2.9.8
```

Comment:
To mirror my suggested wording from above: "[...] for ScanCode 2.9.7 and below." (I'd prefer to write "2.9.**7**" here as that was the version we used before this change, rather that saying "below 2.9.**8**".